service/debugger/locations.go

```diff
@@ -293,6 +294,10 @@ func (loc *NormalLocationSpec) FileMatch(path string) bool {
 	return partialPathMatch(loc.Base, path)
 }
 
+func tryMatchRelativePathByProc(expr, debugname, path string) bool {
+	return path == Path.Join(Path.Dir(debugname), expr)
```

Comment:
Rename `path` to `file` and then `return file == path.Join(filepath.Dir(debugname), expr)`. So we don't have to rename the import of "path".

Also I think this should only be done if `expr[0] == '.'`, it doesn't do anything useful otherwise.